src/java_bytecode/java_bytecode_language.h

```diff
@@ -69,9 +83,14 @@ class java_bytecode_languaget:public languaget
   std::set<std::string> extensions() const override;
 
   void modules_provided(std::set<std::string> &modules) override;
+  virtual void lazy_methods_provided(std::set<irep_idt> &) const;
+  virtual void convert_lazy_method(const irep_idt &id, symbol_tablet &);
 
 protected:
+  bool do_ci_lazy_method_conversion(symbol_tablet &, lazy_methodst &);
```

Comment:
maybe add a comment on what the highly overloaded abbreviation CI means in this context